admin/product.php

```diff
@@ -35,12 +35,25 @@
       }
       break;
     case 'new_product_preview':
+      if (!isset($_POST['master_categories_id'])
+          || ((isset($_POST['products_model']) ? $_POST['products_model'] : '') . (isset($_POST['products_url']) ? implode('', $_POST['products_url']) : '') . (isset($_POST['products_name']) ? implode('', $_POST['products_name']) : '') . (isset($_POST['products_description']) ? implode('', $_POST['products_description']) : '') != '')

```

Comment:
Maybe these checks are excessive? (this 2nd line of concatenated var comes from existing code)